res/values/styles.xml

```diff
@@ -113,4 +113,7 @@
         <item name="colorControlActivated">#ff2090ea</item>
     </style>
 
+    <style name="ShapeAppearanceOverlay.App.CornerSize10Percent" parent="">
+        <item name="cornerSize">10%</item>
```

Comment:
i would use ~ 7% here, so that the _width_ of the curve between the icon and the bubble is roughly always the same; this looks more even.

10% vs 7%:

<img width="284" alt="Screenshot 2023-09-25 at 13 56 48" src="https://github.com/deltachat/deltachat-android/assets/9800740/5c88e3ac-dc3c-4b5b-ae3f-f2e3025833cb"> <img width="284" alt="Screenshot 2023-09-25 at 13 57 13" src="https://github.com/deltachat/deltachat-android/assets/9800740/8b331371-e677-456a-bc5b-e24560329e99">